Userland/Libraries/LibCore/System.h

```diff
@@ -271,4 +271,6 @@ ErrorOr<void> posix_fallocate(int fd, off_t offset, off_t length);
 
 ErrorOr<String> resolve_executable_from_environment(StringView filename, int flags = 0);
 
+char** environment();
```

Comment:
I think it would be nicer for this to return a Span of StringViews, but that would require modifying some other Core::System functions to take that as their env parameter.